ansible/README.md

```diff
@@ -149,6 +149,52 @@ You need to run `initdb` on cloudant **only once** per cloudant database to init
 **Hint:** The `initdb.yml` playbook will only initialize your database if it is not initialized already, else it will skip initialization steps.
 
 
+### Verification after Deployment
+
+After deployment, you can use the `wsk` in your source tree to test if your whisk is operable
+
+- Set up the **API key** for your `wsk`
```

Comment:
Use an enumerated list instead of an unordered list.

Setup **authentication** for the CLI. This command uses the *guest* user, which is automatically added to your OpenWhisk deployment.